assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -538,6 +538,9 @@ che.infra.openshift.trusted_ca.dest_configmap_labels=config.openshift.io/inject-
 # Additional labels to add into every Route created by Che server
 # to allow clear identification.
 che.infra.openshift.route.labels=NULL
+# The hostname that should be used as a suffix for the workspace routes.
```

Comment:
you may need to add empty line above so it does not confuse the script

Reply:
done